numpy/__init__.pyi

```diff
@@ -1086,6 +1086,36 @@ class _ArrayOrScalarCommon:
         where: _ArrayLikeBool_co = ...,
     ) -> _NdArraySubClass: ...
 
+    @overload
+    def argmax(
+        self,
+        axis: None = ...,
+        out: None = ...,
+        *,
+        keepdims: bool = ...,
+        initial: _ScalarLike_co = ...,
+        where: _ArrayLikeBool_co = ...,
+    ) -> intp: ...
```

Comment:
No need for any additional overloads here (or anywhere in this PR), as neither `initial` nor `where` affect the output type, dtype or shape. In this case you can simply add the new parameters to the existing overloads:
``` diff
    @overload
    def argmax(
        self,
        axis: None = ...,
        out: None = ...,
        *,
        keepdims: bool = ...,
+       initial: _ScalarLike_co = ...,
+       where: _ArrayLikeBool_co = ...,
    ) -> intp: ...
```